### ISSUE TITLE
Typo and hard to follow

### ISSUE THREAD

**Issue by venturewise** [AdobeDocs/experience-manager-65.en#268]
Issue in ./help/sites-deploying/indexing-via-the-oak-run-jar.md

See the pink for the typo of "_methion_", but also is there a better way to differentiate the two ways of indexing for new users?
I underlined in green "**online-reindexing of TarMK** is faster than **online TarMK reindexing**..." in my attached screenshot.
![image](https://user-images.githubusercontent.com/3883795/128921036-dee6df8d-f130-44bc-9153-74c17b179128.png)

**Comment by bohnertchris** [comment]
Thank you for reporting, @venturewise. We will review.

**Comment by bohnertchris** [comment]
@adobe export issue to Jira project CQDO

**Comment by bohnertchris** [comment]
@adobe export issue to Jira project CQDOC

**Comment by bohnertchris** [comment]
@adobe export issue to Jira project CQDOC

**Comment by aheim0** [comment]
Thanks for highlighting this. Updates have been made - we hope they help.
Closing this issue now.
